test/e2e/clusterctl_upgrade_test.go

```diff
@@ -325,3 +326,31 @@ var _ = Describe("When testing clusterctl upgrades using ClusterClass (v1.7=>cur
 		}
 	})
 })
+
+var _ = Describe("When testing clusterctl upgrades using ClusterClass (v1.7=>current) on latest management cluster [ClusterClass]", func() {
```

Comment:
This test would now run on e2e-mink8s and on full e2e.

Maybe we want to add some GINKGO_SKIP/GINKGO_FOCUS to only run it on full e2e.

cc @sbueringer

Reply:
Given that it should just work I would keep it simple and just run it in both jobs